tests/preprocessing/test_normalization.py

```diff
@@ -87,12 +89,35 @@ def test_vars_checks(adata_to_norm):
         ep.pp.scale_norm(adata_to_norm, vars=["String1"])
 
 
-@pytest.mark.parametrize("array_type", ARRAY_TYPES)
-def test_norm_scale(array_type, adata_to_norm):
+# TODO: list the supported array types centrally?
```

Comment:
These two lists are not used right now. Should we have a central lookup for which function works with which array type?

Right now, the "ground truth" is tested in the `test_<normalization-method>_array_types` test.

Reply:
> Should we have a central lookup for which function works with which array type?

wdym with lookup here?

Reply:
For example a `supported_array_types` dict or dataframe in e.g. `ehrapy.core._supported_array_types.py`
```
supported_array_types = {
    `ep.pp.scale_norm`: {"np.array": True, "dask.array.Array": True, "sp.sparse.spmatrix: False},
    ....
}
```

Could be used in the test parametrization, instead of writing them up individually all the time.

But I have not seen something like that anywhere before, and there might be good reasons to not do this

Reply:
Very good point. In favor of using dynamically constructed registries indeed.